### PR TITLE
Adding docs to the extension

### PR DESCRIPTION
**Pull request by AnuGayan**

### CODE REVIEW

**Review by suhothayan — CHANGES_REQUESTED** [review]
The licence should be GPL and not Apache

File: docs/images/siddhi-logo-w.svg [hunk 1]
```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
```
Comment (suhothayan) [inline docs/images/siddhi-logo-w.svg:2]: Change this too

File: docs/images/siddhi-logo.svg [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
```
Comment (suhothayan) [inline docs/images/siddhi-logo.svg]: Change this too

File: docs/javascripts/extra.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+/*
+~   Copyright (c) WSO2 Inc. (http://wso2.com) All Rights Reserved.
+~
+~   Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (suhothayan) [inline docs/javascripts/extra.js]: Change this

File: docs/stylesheets/extra.css [hunk 4]
```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
```
Comment (suhothayan) [inline docs/stylesheets/extra.css:4]: Change this

File: docs/license.md [hunk 5]
```diff
@@ -0,0 +1,699 @@
+Copyright (c) 2017 WSO2 Inc. (<http://www.wso2.org>) All Rights Reserved.
+
+WSO2 Inc. licenses this file to you under the Apache License,
```
Comment (suhothayan) [inline docs/license.md:3]: Change this

File: docs/stylesheets/extra.css (hunk 4 above)
Comment (AnuGayan) [reply docs/stylesheets/extra.css:4]: Done

File: docs/license.md (hunk 5 above)
Comment (AnuGayan) [reply docs/license.md:3]: Done

File: docs/images/siddhi-logo-w.svg (hunk 1 above)
Comment (AnuGayan) [reply docs/images/siddhi-logo-w.svg:2]: Done

**Review by suhothayan — APPROVED** [review]

**Pull request merged**